tasks/agent.py

```diff
@@ -146,6 +146,7 @@ def build(
     bundle_ebpf=False,
     agent_bin=None,
     run_on=None,  # noqa: U100, F841. Used by the run_on_devcontainer decorator
+    no_glibc=False,
```

Comment:
This can be `glibc=True`, pyinvoke handles the `no-` prefix for bool arguments automatically.

Reply:
That's nice!